### ISSUE TITLE
Cannot use http client for system database: UNWIND is unsupported.

### ISSUE THREAD

**Issue by wgevaert** [neo4j-php/neo4j-php-client#106]
When trying to write an administrative script, this package gave me the following error while testing against `http://localhost:7474?database=system` with authentication disabled:
```txt
The following unsupported clauses were used: UNWIND. 
The system database supports a restricted set of Cypher clauses. The supported clause structure for procedure calls is: CALL, YIELD, RETURN. YIELD and RETURN clauses are optional. The order of the clauses is fix and each can only occur once.
```

This is caused by the following code inside `Http\HttpConnectionPool`:
```cypher
CALL dbms.components()
YIELD name, versions, edition
UNWIND versions AS version
RETURN name, version, edition
```

Some testing revealed that the following changes make everything work nicely:
- Don't use the `UNWIND` command:
```cypher
CALL dbms.components()
YIELD name, versions, edition
RETURN name, versions, edition
```
- Manually unwind the results a few lines lower: Change all occurences of `$results[0]['version']` to `$results[0]['versions'][0]`.

Also, it seems `BoltConnectionPool` has similar code, maybe the change should be done there as well, for consistency?

**Comment by transistive** [comment]
Also make sure you are on version 2.4.0, IIRC it doesn't get used there anymore on Bolt

**Comment by transistive** [comment]
Fixed in version 2.4.1!
